main.nf

```diff
@@ -781,8 +604,9 @@ process BuildBWAindexes {
     when: !(params.bwa) && params.fasta && 'mapping' in step
 
     script:
+    aligner = params.aligner == "bwa-mem2" ? "bwa-mem2" : "bwa"
```

Comment:
Running with aligner=bwa-mem2 and bwa=false causes this error:


```
Command error:
 [bwa_index] Pack FASTA... 67.78 sec
  * Entering FMI_search
  init ticks = 230721761818
  ref seq len = 6434693834
  binary seq ticks = 134314040444
  build index ticks = 6906133030365
  Allocation of 11.99 GB for cp_occ failed.
  Current Allocation = 71.91 GB
  ```

I think this process needs more memory if run with bwa-mem2: https://github.com/bwa-mem2/bwa-mem2/issues/9
In a different test I have tried with 84GB for GRCh38, which worked well. It was also much faster: around 30min, now it took 1.5h to fail

Reply:
Should we fix that now or fix it in `3.0`?

Reply:
I think in the doc you wrote, that this is an experimental feature or sth. along those lines. ( I can't find it right now). But since human samples are probably more often analysed, what about adding this as a comment in the documentation? I don't know how more resources would go with building the old bwa mem index.

Reply:
-> https://nf-co.re/sarek/dev/parameters#aligner